src/tools/analyzer/sss_analyze.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
```

Comment:
This needs to be configurable during build time to allow proper packaging. Python scripts should use %{__python} interpreter. See this [authselect](https://github.com/authselect/authselect/commit/7193e0ced1b8f0b23581c34dfd41e6606d132500#diff-08be2b652ed70d5f578c36d6cbbeffd1412c34ffeaed925f7eefedea18dbc0b8) commit for an example.

Reply:
Thanks, fixed.